doc/src/docs/InputOutputReference/01a-InputOutputReference.md

```diff
@@ -19760,15 +19762,17 @@ where
 
 #### Field: Name
 
-This field represents a unique identifying name in a group referring Generic contaminantSourceAndSinkNames.
+This field represents a unique identifying name.
 
 #### Field: Zone Name
 
 This field signifies the Name of the zone with constant generic contaminant source and sink.
 
-#### Field: Maximum Generation Rate
+#### Field: Design Generation Rate
 
-This field denotes the full generic contaminant maximum generation rate (m<sup>3</sup>/s). The design generation rate is the maximum amount of generic contaminant expected at design conditions. The design value is modified by the schedule fraction (see Field:Generation Schedule Name).
+This field denotes the full generic contaminant design generation rate (m<sup>3</sup>/s). The design generation rate is the maximum amount of generic contaminant expected at design conditions. The design value is modified by the schedule fraction (see Field:Generation Schedule Name).
+
+When the mass generation rate is available, a conversion is required to meet input requirement with volumetric flow rate. This can be accomplished by the mass flow rate divided by the density of generic contaminant.  
```

Comment:
@lgu1234 Suggested change:
"When the mass generation rate is available, the rate must be converted to a volume flow rate. Use the mass flow rate divided by the vapor density of the generic contaminant."  

Similar comment for other places below.